packages/pds/src/api/com/atproto/admin/sendEmail.ts

```diff
@@ -0,0 +1,46 @@
+import { AuthRequiredError, InvalidRequestError } from '@atproto/xrpc-server'
+import { Server } from '../../../../lexicon'
+import AppContext from '../../../../context'
+
+export default function (server: Server, ctx: AppContext) {
+  server.com.atproto.admin.sendEmail({
+    auth: ctx.roleVerifier,
+    handler: async ({ input, auth }) => {
+      if (!auth.credentials.admin && !auth.credentials.moderator) {
+        throw new AuthRequiredError('Insufficient privileges')
+      }
+
+      try {
+        const {
+          content,
+          recipientDid,
+          subject = 'Message from Bluesky moderator',
+        } = input.body
+        const userInfo = await ctx.db.db
+          .selectFrom('user_account')
+          .where('did', '=', recipientDid)
+          .select('email')
+          .executeTakeFirst()
+
+        if (!userInfo) {
+          throw new InvalidRequestError('Recipient not found')
+        }
+
+        await ctx.moderationMailer.send(
+          { content },
+          { subject, to: userInfo.email },
+        )
+        return {
+          encoding: 'application/json',
+          body: { sent: true },
```

Comment:
I noticed in lexicon the response also has a `message` key.  Will that be used here in any situations?

Reply:
see this please https://github.com/bluesky-social/atproto/pull/1312#discussion_r1264212549